docs/cookbook/configuration/disabling-shop-admin-api.rst

```diff
@@ -52,13 +52,16 @@ The part that has to be removed from this file is shown below:
 
 .. code-block:: yaml
 
+    parameters:
+        # sylius.security.shop_regex: "^/(?!admin|api/.*|api$)[^/]++"
```

Comment:
Is it a valid YAML syntax?

```yaml
parameters:
# nothing here
security:
# ...
```

Reply:
Probably something can go wrong with that :/